pkgs/by-name/at/atools/package.nix

```
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, pkg-config
+, stdenv
+, lib
+# Package dependencies
+, libsForQt5
```

```suggestion
, qt5
```
this works too, and it's shorter and it's the way going forward with qt6